libultraship/libultraship/Lib/Fast3D/gfx_pc.cpp

```diff
@@ -2881,7 +2860,7 @@ uint16_t gfx_get_pixel_depth(float x, float y) {
 
     get_pixel_depth_pending.emplace(x, y);
 
-    map<pair<float, float>, uint16_t> res = gfx_rapi->get_pixel_depth(game_renders_to_framebuffer ? game_framebuffer : 0, get_pixel_depth_pending);
+    unordered_map<pair<float, float>, uint16_t> res = gfx_rapi->get_pixel_depth(game_renders_to_framebuffer ? game_framebuffer : 0, get_pixel_depth_pending);
```

Comment:
`gfx_rapi->get_pixel_depth` changes to `unordered_map` for the sake of being able to merge the result into `get_pixel_depth_cached` in the next line. This change would not be necessary if there is an efficient way to merge a `map` into a `unordered-map`.